### PR TITLE
actually good uwuifier

### PR DESCRIPTION
**Pull request by amyulating**
im cool

### CODE REVIEW

File: owoify/index.js [hunk 1]
```diff
@@ -1,32 +1,104 @@
...
+      "(´꒳`)",
+      "♥(。U ω U。)",
+      "(˘ε˘)",
+      "( ˘ᴗ˘ )",
+      "(*ฅ́˘ฅ̀*)",
+      "*screams*",
+      "*twearks*",
+      "*sweats*",
+    ];
+    console.log("shit got loaded");
```
Comment (lexisother) [inline owoify/index.js]: :)
```suggestion

```

File: owoify/index.js [hunk 2]
```diff
@@ -1,32 +1,104 @@
...
+          ismodified = false;
+          //yes its stutter but fuck you im too lazy to fix it
+          let chancesutter = random(1, 5);
+          x = input[i]
+            .toLowerCase()
+            .replace("r", "w")
+            .replace("l", "w")
+            .replace("ne", "nye")
+            .replace("no", "nyo")
+            .replace("ni", "nyi");
```
Comment (lexisother) [inline owoify/index.js:71]: You're not taking into consideration capital letters won't be replaced. See if you can do the same type of replacement for the other three `.replace` invocations.
```suggestion
            .replace(/[rl]/g, "w")
            .replace(/[RL]/g, "W")
            .replace("ne", "nye")
            .replace("no", "nyo")
            .replace("ni", "nyi");
```

**Review by lexisother — CHANGES_REQUESTED** [review]
Couple of sidenotes!

File: owoify/index.js (hunk 2 above)
Comment (amyulating) [reply owoify/index.js:71]: look at line 66, everything is lowercase anyway,

Comment (lexisother) [reply owoify/index.js:71]: Apologies, oversight!

**Review by lexisother — APPROVED** [review]

**Pull request merged**